common/include/pcl/pcl_base.h

```diff
@@ -237,7 +240,7 @@ namespace pcl
       std::vector<int> field_sizes_;
```

Comment:
Let's make another PR for this?

Reply:
Since this is explicitly used for sizes, and thus cannot be negative, do you suppose `uindex_t` would be better for this?

Reply:
I suppose so

Reply:
I'll go ask the compilers.